api/api.go

```diff
@@ -39,14 +39,11 @@ func HttpPOST(url string, data string) (resp *http.Response, err error) {
 // the API.
 func handleHTTPRequest(mux *http.ServeMux, url string, handler http.HandlerFunc) {
 	mux.HandleFunc(url, func(w http.ResponseWriter, req *http.Request) {
-		permittedClients := []string{"Sia-Agent", "Sia-Miner", "Electron", "AtomShell"}
-		for _, pc := range permittedClients {
-			if strings.Contains(req.UserAgent(), pc) {
-				handler(w, req)
-				return
-			}
+		if !strings.Contains(req.UserAgent(), "Sia-Agent") {
```

Comment:
could even be a simple `==` if you want maximum strictness. But I suppose this allows you to do things like `Sia-Agent-Miner` if you want.